include/pgduckdb/pgduckdb_planner.hpp

```diff
@@ -11,6 +11,6 @@ extern "C" {
 
 extern bool duckdb_explain_analyze;
 
-PlannedStmt *DuckdbPlanNode(Query *parse, int cursor_options);
+PlannedStmt *DuckdbPlanNode(Query *parse, int cursor_options, int elevel);
```

Comment:
nit
```suggestion
PlannedStmt *DuckdbPlanNode(Query *parse, int cursor_options, int elevel_on_error);
```

Reply:
I changed this to `bool throw_error` for consistency with `IsAllowedStatement`.